cumulusci/__init__.py

```diff
@@ -2,5 +2,5 @@
 import os
 
 __import__("pkg_resources").declare_namespace("cumulusci")
-__version__ = "2.4.3"
+__version__ = "4"
```

Comment:
Is this change intended? Shouldn't we have "2.4.4"?

Reply:
Ouch, yeah, this is a typo.